internal/handlers/jsonb1/msg_find_or_count.go

```diff
@@ -43,13 +43,28 @@ func (h *storage) MsgFindOrCount(ctx context.Context, msg *wire.OpMsg) (*wire.Op
 	db := m["$db"].(string)
 
 	projection, ok := m["projection"].(types.Document)
+	projectionStr := "_jsonb"
 	if ok && len(projection.Map()) != 0 {
-		return nil, common.NewErrorMessage(common.ErrNotImplemented, "MsgFind: projection is not supported")
+		ks := ""
+		for i, k := range projection.Keys() {
+			if i != 0 {
+				ks += ","
```

Comment:
```suggestion
				ks += ", "
```

The same in other places